src/Pages/VideoPage/index.js

```diff
@@ -1,35 +1,59 @@
+import "./VideoPage.css";
 import React from "react";
 import { useParams } from "react-router-dom";
 import YouTube from "react-youtube";
-import { useVideo } from "../../context/Video/context";
-import "./VideoPage.css";
 import { BiLike } from "react-icons/bi";
-import { MdPlaylistAdd, MdHistory } from "react-icons/md";
+import { MdPlaylistAdd, MdOutlineWatchLater } from "react-icons/md";
+import { LikeVideo, unLikeVideo, isVideoLiked} from "../../context/Video/liked";
+import { useVideo } from "../../context/Video/context";
+import { useAuth } from "../../context/Auth/context";
```

Comment:
you can create an index file and export all context and you have to write only one line then

Reply:
thanks for pointing it out, I have made the required changes.